### PR TITLE
Bug 1951387: Add after deprecation

### PR DESCRIPTION
**Pull request by gallettilance**
Signed-off-by: Lance Galletti <galletti@bu.edu>

https://bugzilla.redhat.com/show_bug.cgi?id=1951387

### CODE REVIEW

File: staging/operator-registry/pkg/registry/populator_test.go [hunk 1]
```diff
@@ -911,6 +911,140 @@ func TestDeprecateBundle(t *testing.T) {
...
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.description, func(t *testing.T) {
+			logrus.SetLevel(logrus.DebugLevel)
+			db, cleanup := CreateTestDb(t)
+			defer cleanup()
+
+			load, err := sqlite.NewSQLLiteLoader(db, sqlite.WithEnableAlpha(true))
```
Comment (timflannagan) [inline staging/operator-registry/pkg/registry/populator_test.go:973]:
```
PASS
coverage: 100.0% of statements
ok  	github.com/operator-framework/operator-registry/pkg/lib/dns	0.055s	coverage: 100.0% of statements
?   	github.com/operator-framework/operator-registry/pkg/lib/graceful	[no test files]
?   	github.com/operator-framework/operator-registry/pkg/lib/image	[no test files]
# github.com/operator-framework/operator-registry/pkg/registry_test [github.com/operator-framework/operator-registry/pkg/registry.test]
pkg/registry/populator_test.go:973:45: undefined: sqlite.WithEnableAlpha
```

**Comment by njhale** [comment]
Spoke w/ @gallettilance -- I'm going to take a crack at fixing this up today.

**Pull request merged**